nitpick.json

```diff
@@ -0,0 +1,6 @@
+{
+  "ignore": [
+    "tests/*",
```

Comment:
Actually, I would rather we don't ignore these folders. I know it means we get namespace complaints everywhere, but I still like getting the PSR-2 sniff even if it means haaving to delete all the namespace complaints manually.